.github/workflows/ci.yaml

```diff
@@ -39,25 +38,55 @@ jobs:
       - name: Clone OpenAssetIO
         uses: actions/checkout@v2
         with:
-          repository: TheFoundryVisionmongers/OpenAssetIO
+          repository: foundrytom/OpenAssetIO
+          ref: work/347-pythonVenvBuild
           path: dependencies/OpenAssetIO
 
+      - name: Cache conan packages
+        id: cache-conan
+        uses: actions/cache@v2
+        with:
+          path: ~/conan
```

Comment:
Ahhhhhh, awesome thanks, I did wonder if that var was a DRY device, or actually also doing Conan things. I like your suggestion, given hopefully all this will go soon.